index.html

```diff
@@ -5,55 +5,55 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Mirco Warnck</title>
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="./styles/main.css" />
   </head>
 
   <body>
-    <header>
+    <header class="header">
       <h1>Mirco Warnck</h1>
-      <nav>
-        <ul>
+      <nav class="nav">
+        <ul class="nav__ul">
           <li>
-            <a class="nav-links" href="#biography">Biography</a>
+            <a class="nav__nav-links" href="#biography">Biography</a>
           </li>
           <li>
-            <a class="nav-links" href="#contact">Contact</a>
+            <a class="nav__nav-links" href="#contact">Contact</a>
           </li>
           <li>
-            <a class="nav-links" href="#networks">Networks</a>
+            <a class="nav__nav-links" href="#networks">Networks</a>
```

Comment:
it´s not wrong but maybe a little bit to much writing ;) i would recommend `class="nav__links"` instead of `class="nav__nav-links"`